### PR TITLE
feat(site-header-core): adding login box, added test - TWIG-144

### PR DESCRIPTION
**Pull request by planctus**
# PR description

Please drop a few lines about the PR: what it does, how to test it, etc.

## QA Checklist

In order to ensure a safe and quick review, please check that your PR follow those guidelines:

- [ ] I have put the vanilla component as `devDependencies`
- [ ] I have put the specs package as `devDependencies`
- [ ] I have added the components directly used in the twig file (with `include` or `embed`) as `dependencies`
- [ ] My component is listed in `@ecl-twig/ec-components`'s `dependencies`
- [ ] My variables naming follow the guidelines (snake case for twig)
- [ ] I have provided tests
- [ ] I have provided documentation (for the "notes" tab)
- [ ] If my local `yarn.lock` contains changes, I have committed it
- [ ] I have given my PR the proper label (`pr: review needed` to indicate that I'm done and now waiting for a review ,`pr: wip` to indicate that I'm actively working on it ...)

### CODE REVIEW

File: src/ec/packages/ec-component-site-header-core/README.md [hunk 1]
```diff
@@ -14,6 +14,16 @@ npm install --save @ecl-twig/ec-component-site-header-core
   - "alt" (string) (default: ''): Logo alt attribute.
   - "href" (string) (default: ''): Logo URL.
   - "src" (string) (default: ''): Logo image file path, eg. dist/images/logo--en.svg.
+- **"login_box"** (associative array) format:
+  - "id": (string) Id of the box
```
Comment (papegaill) [inline src/ec/packages/ec-component-site-header-core/README.md]: remove or move the two points

File: src/ec/packages/ec-component-site-header-core/ecl-site-header-core.html.twig [hunk 2]
```diff
@@ -134,6 +159,59 @@
...
+                type: 'general',
+                name: 'log-in',
+                size: 's'
+              }
+            } only %}
+            {{- login_toggle.label_not_logged -}}
+          </a>
+        {% endif %}
+      {% endif %}
         <a
```
Comment (papegaill) [inline src/ec/packages/ec-component-site-header-core/ecl-site-header-core.html.twig]: Missing closing </div> here that breaks the alignment

File: src/ec/packages/ec-component-site-header-core/README.md (hunk 1 above)
Comment (papegaill) [inline src/ec/packages/ec-component-site-header-core/README.md]: remove or move the two points

File: src/ec/packages/ec-component-site-header-core/site-header-core.story.js [hunk 3]
```diff
@@ -19,6 +19,7 @@ storiesOf('Components/Site Headers/Core', module)
     () => {
```
Comment (papegaill) [inline src/ec/packages/ec-component-site-header-core/site-header-core.story.js]: maybe we need also a "logged in" story like ECL and standardised component? 
Also by checking the site-headers-standardised stories. 
We have problem with de default story, it seems we are always logged in.

**Comment by planctus** [comment]
I don't know what happened, i read your comment and i wanted to tell you to do the change yourself, @papegaill , but i managed to delete your comment, actually :)

File: src/ec/packages/ec-component-site-header-core/site-header-core.story.js (hunk 3 above)
Comment (papegaill) [reply src/ec/packages/ec-component-site-header-core/site-header-core.story.js]: fixed

**Pull request merged**